crates/index-scheduler/src/batch.rs

```diff
@@ -781,7 +780,9 @@ impl IndexScheduler {
                     let index = self.index_mapper.index(&rtxn, name)?;
                     let dst = temp_snapshot_dir.path().join("indexes").join(uuid.to_string());
                     fs::create_dir_all(&dst)?;
-                    index.copy_to_file(dst.join("data.mdb"), CompactionOption::Enabled)?;
+                    index
+                        .copy_to_file(dst.join("data.mdb"), CompactionOption::Enabled)
+                        .map_err(|e| Error::from_milli(e, Some(uuid.to_string())))?;
```

Comment:
```suggestion
                        .map_err(|e| Error::from_milli(e, Some(name.to_string())))?;
```

here it's the name of the index that we want to return